File: project/leogreen_server/src/main/java/at/htl/model/pojo/Game.java
package at.htl.model.pojo;

import at.htl.model.entity.TagEntity;
import lombok.Data;

import javax.persistence.ManyToOne;
import java.io.Serializable;
import java.util.List;

@Data
public class Game implements Serializable {
    Long id;
    Integer state;
    List<User> users;
    Quiz quiz;

    public Game(Long id, Integer state, List<User> users, Quiz quiz) {
        this.id = id;
        this.state = state;
        this.users = users;
        this.quiz = quiz;
    }

    public Game() {
    }
}

File: project/client/leoGreen/src/app/model/game.ts
import {User} from "./user";
import {Quiz} from "./quiz";

export class Game {
  id: number
  state: number
  users: User[]
  quiz: Quiz

  constructor(id: number, state: number, users: User[], quiz: Quiz) {
    this.id = id;
    this.state = state;
    this.users = users;
    this.quiz = quiz;
  }
}
